### ISSUE TITLE
Restart atom not restore opened files (tabs)

### ISSUE THREAD

**Issue by Daijobou** [atom/atom#17095]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

I have create a screen-capture
![missing_project_files](https://user-images.githubusercontent.com/260343/38442320-39cf0400-39e8-11e8-9b41-223369ba01dc.gif)

you see: open atom, add folder, open file, close atom, (white screen) restart atom, tabs are not restored.

### Steps to Reproduce

1. Start a empty atom
2. add "add project folder"
3. open file
4. close atom
5. open atom

**Expected behavior:** [What you expect to happen]

restore opened file (tab)

**Actual behavior:** [What actually happens]

no files(tabs) will be restored

**Reproduces how often:** [What percentage of the time does it reproduce?]

always 100%

### Versions

Windows 10
Atom    : 1.25.1
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0
apm  1.19.0
npm  3.10.10
node 6.9.5 x64
atom 1.25.1
python
git 2.7.0.windows.1
visual studio

**Comment by ashley-hu** [comment]
Hi, 
I replicated the steps you provided but could not reproduce the issue. 
When I reopened Atom, the file that the tab was opened on was restored. 

The file that I made before I closed it.
![screen shot 2018-04-09 at 10 28 34 pm](https://user-images.githubusercontent.com/19430665/38533047-70c2c7da-3c45-11e8-941b-1510b262b6b1.png)

After I reopened Atom, the tab was restored. 
![screen shot 2018-04-09 at 10 28 47 pm](https://user-images.githubusercontent.com/19430665/38533058-87eab076-3c45-11e8-920b-c0f98551e7f3.png)

### Versions
Mac OS 10.11.6
Atom 1.25.1

**Comment by Daijobou** [comment]
That "white screen" is like a desktop background. (I have opened a maximized notepad.exe behind atom). Here is no crash. I open und close atom without a issue, but the tabs are missing.

For me that problem with the tabs is new too. Maybe its related with the other issue I described.

So I have (without uninstall) installed atom 1.25.1 again and its working now out-of-box with tabs. Funfact: its not opened my last project, its opened the project and files from project that was crashed and don't get open.
